client/routes.js

```diff
@@ -51,15 +54,18 @@ const mapState = (state) => {
   return {
     // Being 'logged in' for our purposes will be defined has having a state.user that has a truthy id.
     // Otherwise, state.user will be an empty object, and state.user.id will be falsey
-    isLoggedIn: !!state.user.id
+    isLoggedIn: !!state.user.id,
+    isAdmin: !state.user.isAdmin
```

Comment:
Should this not have a bang? The logic seems to return false, if the isAdmin boolean is true